test/iterator_range.cpp

```diff
@@ -66,8 +65,6 @@ int main()
     CHECK(l0.end() == li.end());
     CHECK(l0.size() == li.size());
 
-    l0 = view::all(li);
-
```

Comment:
Why?

Reply:
This ought to work; maybe `sized_iterator_range` doesn't have the same implicit conversions as in cmcstl2/P0789? I'll investigate.

Reply:
Yip - that's it. We need to port `subrange` to range-v3.

Reply:
Would you like me to create a reminder issue that we need to port `subrange`, or block this merge until that happens, or...?

Reply:
You could patch this up by adding implicit conversions from `ref_view` to `[sized_]iterator_range`.